kotlin/services/kendra/src/main/kotlin/com/example/kendra/DeleteIndex.kt

```diff
@@ -0,0 +1,59 @@
+//snippet-sourcedescription:[DeleteIndex.kt demonstrates how to delete an Amazon Kendra index.]
+//snippet-keyword:[SDK for Kotlin]
+//snippet-keyword:[Code Sample]
+//snippet-service:[Amazon Kendra]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/10/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.kendra
+
+// snippet-start:[kendra.kotlin.delete.index.import]
+import aws.sdk.kotlin.services.kendra.KendraClient
+import aws.sdk.kotlin.services.kendra.model.DeleteIndexRequest
+import kotlin.system.exitProcess
+// snippet-end:[kendra.kotlin.delete.index.import]
+/**
+To run this Kotlin code example, ensure that you have set up your development environment,
+including your credentials.
+
+For information, see this documentation topic:

```

Comment:
For more information, see the following documentation topic:

Reply:
fixed